be/src/storage/rowset/bitmap_index_writer.cpp

```diff
@@ -173,6 +171,13 @@ class BitmapIndexWriterImpl : public BitmapIndexWriter {
     uint64_t size() const override {
```

Comment:
How about the call frequency of this function?
If the frequency is low, _size_changed is non-sense.

Reply:
This function is called every we append a chunk， we need to check if the all chunk size in a segment is greater than the threshold. If it is, we need to flush the chunk.
We can omit the _size_changed here.